src/controllers/update/overseas.entity.update.details.controller.ts

```diff
@@ -68,6 +68,8 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const session = req.session as Session;
     const entityName = req.body[EntityNameKey];
 
+    setOriginalIncorporationCountry(session, data);
```

Comment:
Worth a comment here to say why this is done. It's not done for any of the other fields. I think it's because it's a read-only field but there's a risk it's overwritten to due to auto-fill function in some browsers?

Reply:
Do you have a suggestion for the comment?

Reply:
// The original incorporation country is set again here in case it's been overwritten with an incorrect value (it's the only read-only field on the screen and may be replaced with another value by browser auto-fill functionality)

Assuming what I've written above is correct!